plugins/package-managers/node/src/main/kotlin/Yarn2.kt

```diff
@@ -266,13 +266,13 @@ class Yarn2(
         )
 
         // First pass: Parse the headers and query each package details.
-        val packagesHeaders: Map<String, PackageHeader>
+        val packageHeaders: Map<String, PackageHeader>
```

Comment:
Unrelated, but if you feel like additionally changing this unusual syntax to

    val packageHeaders = mutableMapOf<String, PackageHeader>()

that'd be great.

Reply:
It will go away in my next PR anyway.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/9116/commits/0e152b732ef26a15fb9d97d839fd93d8c00e8d3d#diff-42aedc605f6889312d50555012975d3e352996ce23727883d2729757acaaee78L268